IceHUD.lua

```diff
@@ -36,6 +36,7 @@ else
 end
 
 -- compatibility/feature flags
+IceHUD.GetPlayerAuraBySpellID = IceHUD.WowMain and IceHUD.WowVer > 90000
```

Comment:
Let's check for the feature (C_UnitAuras and the function, probably) instead of a version number. That way if/when a Classic client adds support, this automatically works.

Reply:
Just updated that line. I'm new to LUA, so let me know if there's a better way to write any of this.